src/Build/Logging/BinaryLogger/BuildEventArgsReader.cs

```diff
@@ -90,6 +103,21 @@ public BuildEventArgs Read()
             return result;
         }
 
+        /// <summary>
+        /// For now it's just the ProjectImportArchive.
+        /// </summary>
+        private static bool IsBlob(BinaryLogRecordKind recordKind)
+        {
+            return recordKind == BinaryLogRecordKind.ProjectImportArchive;
```

Comment:
We'll deal with it then ;) We can convert this to a switch statement or anything else.